src/pages/Editor/InternalEditor.tsx

```diff
@@ -170,6 +170,19 @@ const InternalEditor = ({ values }: {
     return zipObject(filteredCustomAttributes, Array(filteredCustomAttributes.length).fill(''));
   };
 
+  const revertMergeTags = (content: string, generateMergeTag: (s: string) => string) => {
+    const container = document.createElement('div');
+    container.innerHTML = content;
+    container.querySelectorAll('.easy-email-merge-tag-badge').forEach((item: any) => {
+      item.parentNode?.replaceChild(
+        document.createTextNode(generateMergeTag(item.textContent)),
```

Comment:
@saxenasahilas - can we do this:
```tsx
document.createTextNode(`{{${ item.textContent }}}`),
```

Instead of what we have?
The reason is, we will need to use the transformed content before (at `const customAttributes = onlyGetUsedCustomAttributes(values);`) where we are finding used attributes.